spec/mongoid/configuration_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe Mongoid::Enum::Configuration do
+  subject { Mongoid::Enum::Configuration.new }
+
+  describe "field_name_prefix" do
+    it "has '_' as default value" do
+      expect(subject.field_name_prefix).to eq '_'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.